README.md

```diff
@@ -20,32 +20,28 @@ import (
 
 func main() {
 	// Generate token that is valid for 60 seconds using the default secret
-	var token string
-	var err error
-	if token, err = ft.GenerateToken("RmFzdGx5IFRva2VuIFRlc3Q=", 60*time.Second, base64.StdEncoding); err != nil {
-		log.Fatal("Error while generating token", err)
-	}
+	token := ft.GenerateToken("RmFzdGx5IFRva2VuIFRlc3Q=", 60*time.Second, base64.StdEncoding); err != nil {
```

Comment:
I hadn't noticed the error was always nil. I completely agree about removing it 😄